models/models.go

```diff
@@ -100,6 +101,7 @@ func NewTestEngine(x *xorm.Engine) (err error) {
 		return fmt.Errorf("connect to database: %v", err)
 	}
 
+	x.SetMapper(core.GonicMapper{})
```

Comment:
Nope, not at all. It only maps names with two successive Uppercase characters different and there are no structs or fields in models/ that have that.